server/views/map/show.jade

```diff
@@ -1,35 +1,41 @@
 extends ../layout-wide
 block content
-        .text-center.map-fixed-header.asterisk-explanation
+        .text-center.map-fixed-header
             p Challenges required for certifications are marked with a *
             .row.map-buttons
                 button.center-block.btn.btn-sm.btn-block.btn-primary.active#showAll Collapse all challenges
+            .row.map-buttons
+                .input-group
+                    input#map-filter.form-control(type="text" placeholder="challenge name")
```

Comment:
It says "Search term or @username".  This says "challenge name".  I see them as being semantically identical.